inductiva/fluids/scenarios/wind_tunnel/wind_tunnel.py

```diff
@@ -0,0 +1,145 @@
+"""Wind tunnel scenario to run an object over an air flow."""
+
+from functools import singledispatchmethod
+import os
+import shutil
+from typing import Optional, List
+
+from absl import logging
+
+from inductiva.types import Path
+from inductiva.scenarios import Scenario
+from inductiva.simulation import Simulator
+from inductiva.fluids.simulators import OpenFOAM
+from inductiva.utils.templates import batch_replace_params_in_template
+from inductiva.utils.files import remove_files_with_tag
+
+OPENFOAM_TEMPLATE_INPUT_DIR = "openfoam_template"
+
+
+class WindTunnel(Scenario):
+    """Physical scenario of a general wind tunnel simulation."""
```

Comment:
This needs a bit more information. Where does the wind blow from? -x to +x? Is it configurable?